src/components/Pagination/index.js

```diff
@@ -68,12 +68,12 @@ class Pagination extends React.Component {
   lastPageLink( pager ) {
     return pager.isLastPage() ?
       <li className="disabled">
-        <a href="#" aria-label="Next">
+        <a href="#" aria-label="Next" onClick={ function(e){ e.preventDefault(); } }>
           <span aria-hidden="true">{ this.props.labels.last }</span>
         </a>
       </li> :
       <li onClick={ this.setPage.bind( this, pager.total - 1 ) }>
-        <a href="#" aria-label="Next">
+        <a href="#" aria-label="Next" onClick={ function(e){ e.preventDefault(); } }>
           <span aria-hidden="true">{ this.props.labels.last }</span>
         </a>
       </li>;
```

Comment:
Real link sounds like a very interesting option. I'm thinking it could also be very crawler friendly :)